packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/DynamicTable/TableRows/index.js

```diff
@@ -22,11 +22,14 @@ const TableRows = ({ headers, rows, onModalToggle }) => {
               fn: () => onModalToggle(data.id),
             })}
           >
-            {headers.map(({ key, name }) => {
+            {headers.map(({ key, name, cellFormatter }) => {
               return (
                 <Td key={key}>
                   <Typography textColor="neutral800">
-                    <CellValue type={key} value={data[name]} />
+                    <CellValue
+                      type={key}
+                      value={cellFormatter ? cellFormatter(data[name]) : data[name]}
```

Comment:
I agree that things should be done in 1 place. I think CellValue is currently a component because it uses 2 react hooks, so I'm not sure we could easily convert it to a function though

Reply:
Couldn't the hooks be moved inside this `TableRows` component and then used inside a function?